internal/cmd/root.go

```diff
@@ -135,8 +135,12 @@ func openPostgreSQLDatabase(ctx context.Context) (database.DB, error) {
 	user, _ := os.LookupEnv("SHIORI_PG_USER")
 	password, _ := os.LookupEnv("SHIORI_PG_PASS")
 	dbName, _ := os.LookupEnv("SHIORI_PG_NAME")
+	sslmode, ok := os.LookupEnv("SHIORI_PG_SSLMODE")
```

Comment:
Keep in mind that in this case `ok` will be `true` in the case `SHIORI_PG_SSLMODE` is an empty string. Unsure how Posgres will react to that. Maybe is better to check for `sslMode == ""`.